server/src/Validation/Auth.validation.js

```diff
@@ -47,21 +46,21 @@ export default class AuthValidation {
       });
     }
 
-    if (!validEmail.test(email)) {
+    if (!validateEmail(email)) {
```

Comment:
Please change this to something like `isValidEmail`. So that it is not a verb.
```suggestion
    if (!isValidEmail(email)) {
```

Reply:
done...